examples/glamorous-jest-react/index.js

```diff
@@ -1,19 +1,20 @@
 import React from 'react'
-import * as glamorous from 'glamorous'
+/* eslint-disable */
+import * as glamorous from 'glamorous';
```

Comment:
Shouldn't this be `import glamorous from 'glamorous'`? `glamorous` only has one export.